Automattic/jetpack
# Setup Wizard: Remove setup wizard

#### Changes proposed in this Pull Request:

This PR removes the Setup Wizard, which has been replaced by the Recommendations tab.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. Start a new site and use the snippets plugin to add this, which would activate the Setup Wizard:
```
add_filter( 'jetpack_show_setup_wizard', '__return_true' );
add_filter( 'jetpack_pre_connection_prompt_helpers', '__return_true' );
```
2. Verify that the "Set up" option is not available in the left side nav bar.
3. Connect Jetpack and click around through various pages. Check the error log to make sure that there are no errors.


#### Proposed changelog entry for your changes:


None needed (the setup wizard was tested only on Bluehost sites and was replaced by the Recommendations tab which is showing for all users)
